tasks/gh-pages.js

```diff
@@ -0,0 +1,241 @@
+const git = require('simple-git');
+const semv = require('semver');
+const file = require('fs');
+const path = require('path');
+const ZERO = '0.0.0';
+const USER = process.env.GH_USER_NAME;
+const PASS = process.env.GH_ACCESS_TOK;
+const REPO = 'github.com/Tradeshift/tradeshift-ui.git';
+const log = msg => console.log(msg);
+const tojson = object => JSON.stringify(object, 0, 2);
+const abspath = (...paths) => path.join(__dirname, ...paths.map(String));
+const encoding = read => (read ? 'UTF-8' : { encoding: 'UTF-8' });
+const readfile = where => file.readFileSync(where, encoding(true));
+const writefile = (data, where) => file.writeFileSync(where, data, encoding());
+const nukefolder = folder => rimraf(abspath(folder));
+const getpackage = where => JSON.parse(readfile(abspath(where)));
```

Comment:
`getpackage` -> `getpckg`,
`setpackage` -> `setpckg`?

Reply:
Ah. I get it. Right, the secret amount of letters to use for optimal performance. Sure. Will update!